test/parallel/test-buffer-bytelength.js

```diff
@@ -23,8 +23,6 @@ const vm = require('vm');
   );
 });
 
-assert.strictEqual(Buffer.byteLength('', undefined, true), -1);
```

Comment:
Why is the test being removed? Do we not still want to make sure the behavior is unchanged?

Reply:
Oh, I see, there is no third argument. Never mind, I get it.